src/renderer/theme.tsx

```diff
@@ -20,16 +20,19 @@ const theme = responsiveFontSizes(
       },
     },
     typography: {
+      body1: {
+        color: colors.grey[300],
+      },
       fontFamily: ['Rubik', 'sans-serif'].join(','),
-      'h-100': {
+      h1: {
         fontSize: '24px',
-        lineHeight: '36px',
+        lineHeight: 36 / 18, // 36px
```

Comment:
Just curious, what is the benefit of this?

Reply:
The alternative throws an error lol. Try it

Reply:
yeah weird. thanks for the fix